src/octopeer-github/test/main/Options/OptionsTest.ts

```diff
@@ -47,51 +48,23 @@ describe("The Options class", function() {
         expect(spyNotify).not.toHaveBeenCalled();
     });
 
-    it("should get the logging value", function() {
-        expect(Options.getLogging()).toBe(true);
+    it("should get the option value", function() {
+        let optionList: string[] = Options.generateOptionList();
+        // HashFile is false by default.
+        let turnedOff = 7;
+        for (let i = 0; i < optionList.length && i !== 7; i++) {
```

Comment:
You let the for-loop stop at 7? If `hashFile` is false by default, I think the dnwOptions should also still be tested.